index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Hello World</title>
+</head>
+<body>
+    <h1>Hello World</h1>
```

Comment:
- [ ]  Please add a class for example `<h1 class="header">Hello Microverse</h1>` to the `h1` tag in order to have access to the class in your CSS file to give some stylings.